lib/ansible/modules/network/ironware/ironware_command.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: ironware_command
+version_added: "2.5"
+author: "Paul Baker (@paulquack)"
+short_description: Run arbitrary commands on Brocade IronWare devices
+description:
+  - Sends arbitrary commands to a Brocade Ironware node and returns the results
+    read from the device. The C(ironware_command) module includes an
```

Comment:
Remove the module name
Mention the option and format correctly http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options

`This module includes an I(wait_for) argument that will cause the module to wait for a specific condition....`